File: bluetooth_debugger.py
# navigate to the project directory in Terminal,
# and run command "sudo python bluetooth_debugger.py".

import os

path = "/Library/Preferences/"

os.chdir(path)

for i in os.listdir(path):
    if (i == 'com.apple.Bluetooth.plist'):
        os.unlink(i)
    else:
        pass

File: bluetoothDebugger.js
/*
navigate to the project directory in Terminal,
and run command `sudo node bluetoothDebugger.js`.
*/

const fs = require("fs")

const path = "/Library/Preferences/"
const corruptFile = "com.apple.Bluetooth.plist"

const absolutePath = `${path}${corruptFile}`
console.log(absolutePath)

if (fs.existsSync(absolutePath)) {
    fs.unlink(absolutePath, err => {
        if (err) {
            return console.log(err)
        }
        console.log("file deleted. Restart your computer")
    })
}
